tests/scripts/test_make_release.py

```diff
@@ -17,6 +22,7 @@
 COMMITS__VERSION_4_0_1 = DATA_PATH.joinpath("commits__version_4_0_1.json")
 
 
+pytest.importorskip("make_release")
 def test_get_git_commits_since_tag():
     with open(COMMITS__VERSION_4_0_1, mode="rb") as commits, \
             mock.patch("urllib.request.urlopen", side_effect=lambda n: commits), \
```

Comment:
I have some problem to understand this function being called out of the function. shouldn't it be like this ?:
```suggestion
def test_get_git_commits_since_tag():
    pytest.importorskip("make_release") 
    with open(COMMITS__VERSION_4_0_1, mode="rb") as commits, \
            mock.patch("urllib.request.urlopen", side_effect=lambda n: commits), \
```
and not try to import make_release at the top ?

Reply:
You are correct. Trying to import `make_release` at the top should not really hurt here nevertheless and avoids having to re-import this module in each test. Python already attempts to optimize this by only importing each module name once.

Reply:
I decided to actually use the return value, which avoids our exception handling completely.